org.zrythm.Zrythm.json

```diff
@@ -0,0 +1,442 @@
+                    "url": "https://gitlab.gnome.org/GNOME/libadwaita.git",
+                    "tag": "1.0.1",
+                    "commit": "b274a0c32c51a92692e3b282f04e88f033ba73b1"
+                }
+            ],
+            "cleanup": [
+                "/include",
+                "/lib/pkgconfig"
+            ],
+            "modules": [
+                {
+                    "name": "pango",
+                    "buildsystem": "meson",
+                    "config-opts": [
+                        "-Dtests=false",
+                        "-Dbuild-documentation=false"
+                    ],
+                    "sources": [
+                        {
+                            "type": "git",
+                            "url": "https://gitlab.gnome.org/GNOME/pango.git",
+                            "tag": "1.50.3",
+                            "commit": "26aadb2508f9022cbfc72e73b558c6791f5d46d9"
+                        }
+                    ],
+                    "cleanup": [
+                        "/include",
+                        "/lib/debug",
+                        "/bin/pango-*"
+                    ]
+                },
+                {
+                    "name": "libsass",
+                    "buildsystem": "meson",
+                    "cleanup": [
+                        "*"
+                    ],
+                    "sources": [
+                        {
+                            "type": "git",
+                            "url": "https://github.com/lazka/libsass.git",
+                            "branch": "meson",
+                            "commit": "302397c0c8ae2d7ab02f45ea461c2c3d768f248e"
+                        }
+                    ]
+                },
+                {
+                    "name": "sassc",
+                    "buildsystem": "meson",
+                    "cleanup": [
+                        "*"
+                    ],
+                    "sources": [
+                        {
+                            "type": "git",
+                            "url": "https://github.com/lazka/sassc.git",
+                            "branch": "meson",
+                            "commit": "82803377c33247265d779af034eceb5949e78354"
+                        }
+                    ]
+                },
+                {
+                    "name": "gtk",
+                    "buildsystem": "meson",
+                    "config-opts": [
+                        "-Dbuild-documentation=false",
+                        "-Dintrospection=disabled",
+                        "-Dgtk_doc=false",
+                        "-Dtests=false",
+                        "-Dexamples=false",
+                        "-Dvapi=false"
+                    ],
+                    "sources": [
+                        {
+                            "type": "git",
+                            "url": "https://gitlab.gnome.org/GNOME/gtk.git",
+                            "tag": "4.6.0",
```

Comment:
does it really require a more recent Gtk4 than the one in the runtime?

Reply:
yes, some symbols were missing in previous releases. 4.6.0 also has a few important bug fixes